src/Symfony/Component/Process/PhpExecutableFinder.php

```diff
@@ -37,7 +37,14 @@ public function find($includeArgs = true)
     {
         if ($php = getenv('PHP_BINARY')) {
             if (!is_executable($php)) {
-                return false;
+                $command = '\\' === \DIRECTORY_SEPARATOR ? 'where' : 'command -v';
```

Comment:
IIRC, `where` can return several lines

Reply:
According to https://ss64.com/nt/where.html, it's true. Now, I always use the first line.